manager/integration/tests/test_upgrade.py

```diff
@@ -162,23 +188,31 @@ def test_upgrade(longhorn_upgrade_type,
     host_id = get_self_host_id()
     pod_data_path = "/data/test"
 
-    pod_volume_name = generate_volume_name()
-
-    auto_salvage_setting = client.by_id_setting(SETTING_AUTO_SALVAGE)
-    setting = client.update(auto_salvage_setting, value="false")
-
-    assert setting.name == SETTING_AUTO_SALVAGE
-    assert setting.value == "false"
-
-    # Create Volume attached to a node.
-    volume1 = create_and_check_volume(client,
-                                      volume_name,
-                                      size=SIZE)
-    volume1.attach(hostId=host_id)
-    volume1 = wait_for_volume_healthy(client, volume_name)
-    volume1_data = write_volume_random_data(volume1)
+    # Disable Auto Salvage Setting
+    update_setting(client, SETTING_AUTO_SALVAGE, "false")
```

Comment:
Why disabling auto salvage?

Reply:
> Why disabling auto salvage?

Hi @innobead :
    After checked the [commit history](https://github.com/longhorn/longhorn-tests/commit/76ff25a5319702746067ab5c20c0e66e89e7fd76), it's due to [issue1384]( https://github.com/longhorn/longhorn/issues/1384#issuecomment-632364661)